content/articles/add_docs.md

```diff
@@ -20,14 +20,14 @@ Our site is a static site hosted somewhere out there on the Internet. Super myst
 
 The site is powered by Pelican, which is a static site generator written in Python. If you have Python on your machine and you want to see what your article looks like on your local machine, install Pelican by typing in your terminal 
 
-    pip install pipenv # if you don't have pipenv on your machine already
-    pipenv install
+    pip install -r requirements.txt
```

Comment:
we don't have a requirements.txt right now.

I would rather do that in a separate pr, and have more generic requirements based on what the application uses, instead of hard coding them to a specific release version.